File: 1313. Decompress Run-Length Encoded List/solution.py
class Solution:
    def decompressRLElist(self, nums: List[int]) -> List[int]:
        result = []
        for index in range(0, len(nums), 2):
            freq = nums[index]
            value = nums[index + 1]
            for episodes in range(freq):
                result.append(value)
        return result

File: 1313. Decompress Run-Length Encoded List/solution.go
func decompressRLElist(nums []int) []int {
	result := []int{}
	for i := 0; i < len(nums); i += 2 {
		var freq = nums[i]
		var value = nums[i+1]
		for j := 0; j < freq; j++ {
			result = append(result, value)
		}
	}
	return result
}
